src/app/styles.tsx

```diff
@@ -0,0 +1,145 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+
+export const PageContainer = styled.div`
+  width: 100%;
+  min-height: 100vh;
```

Comment:
nit: let's use `100svh` so it doesn't get covered by the bottom tab of the mobile browser